tests/basic/fencing/afr-lock-heal-basic.t

```diff
@@ -55,19 +72,16 @@ TEST kill -SIGUSR1 $client1_pid
 EXPECT "Y" is_gfapi_program_alive $client1_pid
 
 # Check lock is present on brick-1 and brick-2
-b1_sdump=$(generate_brick_statedump $V0 $H0 $B0/${V0}0)
-c1_lock_on_b1="$(egrep "$inode" $b1_sdump -A3| egrep 'ACTIVE.*client-0'| uniq| awk '{print $1,$2,$3,S4,$5,$6,$7,$8}'|tr -d '(,), ,')"
-b2_sdump=$(generate_brick_statedump $V0 $H0 $B0/${V0}1)
-c1_lock_on_b2="$(egrep "$inode" $b2_sdump -A3| egrep 'ACTIVE.*client-1'| uniq| awk '{print $1,$2,$3,S4,$5,$6,$7,$8}'|tr -d '(,), ,')"
+EXPECT_WITHIN $PROCESS_UP_TIMEOUT "success" fill_lock_info c1_lock_on_b1 $B0/${V0}0
+EXPECT_WITHIN $PROCESS_UP_TIMEOUT "success" fill_lock_info c1_lock_on_b2 $B0/${V0}1
 TEST [ "$c1_lock_on_b1" == "$c1_lock_on_b2" ]
 
 # Restart brick-3 and check that the lock has healed on it.
 TEST $CLI volume start $V0 force
 EXPECT_WITHIN $PROCESS_UP_TIMEOUT "1" brick_up_status $V0 $H0 $B0/${V0}2
 TEST sleep 10 #Needed for client to re-open fd? Otherwise client_pre_lk_v2() fails with EBADFD for remote-fd. Also wait for lock heal.
```

Comment:
Is this still needed ?

Reply:
@xhernandez  I did indeed think of removing it but I though sleeping for some time would be better than taking multiple state dumps, since without sleep,  it will need more than 1 iteration (and therefore cpu cycles) of statdeump generation. 

But I guess if it helps the test to complete faster even if we take more statedumps, it  is better. I will remove it.

Reply:
I'll wait for the regression results

Reply:
@xhernandez @pranithk  Please merge if it looks good.